### PR TITLE
Fix grammar in admin area labels when no labels exist

### PR DESCRIPTION
**Pull request by aaronsnyder**
Spoke to @dblessing in IRC re: this change. Noticed a grammar error when using GitLab, changed "...no any..." to "...not any...".

### CODE REVIEW

File: app/views/admin/labels/index.html.haml [hunk 1]
```diff
@@ -12,5 +12,5 @@
     = paginate @labels, theme: 'gitlab'
   - else
     .light-well
-      .nothing-here-block There are no any labels yet
-      
\ No newline at end of file
+      .nothing-here-block There are not any labels yet
```
Comment (stanhu) [inline app/views/admin/labels/index.html.haml]: Perhaps it should be: "There are no labels yet"?

**Comment by aaronsnyder** [comment]
Hi @stanhu, thanks for the feedback! You're right, after grepping around a bit, the standard seems to be 'there are no...' over 'there are not any'. My apologies for the oversight! 
 
I don't rebase often enough, and had trouble getting this down to a single commit. Should I nuke this branch, start over, and create a new PR?

**Comment by stanhu** [comment]
@aaronsnyder It's up to you. I think you should be able to squash this into a single commit:

http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

**Comment by aaronsnyder** [comment]
Hi @stanhu, you were right, I was able to squash the commits. Please let me know if there is anything else you need from me.

**Comment by stanhu** [comment]
Thanks, @aaronsnyder!

**Pull request merged**